index.js

```diff
@@ -251,6 +251,32 @@ const execa = (command, args, options) => {
 		return Promise.reject(error);
 	}
 
+	const originalKill = spawned.kill.bind(spawned);
+	spawned.kill = (signal, options) => {
+		const killResult = originalKill(signal);
+		if (signal === undefined ||
+			(typeof signal === 'string' && signal.toUpperCase() === 'SIGTERM') ||
```

Comment:
I don't think we should silently accept lowercase `sigterm`. If anything, we should throw an error that it's incorrect.

Reply:
That was because that's how Node.js `childProcess.kill()` [handles it](https://github.com/nodejs/node/blob/908292cf1f551c614a733d858528ffb13fb3a524/lib/internal/util.js#L225).